### PR TITLE
mise: add page

### PR DESCRIPTION
**Pull request by MustCodeAl**
add tldr for mise runtime

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **2024.2.18**

### CODE REVIEW

**Review by gutjuri — APPROVED** [review]
Thanks!

File: pages/common/mise.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# mise
+
+> Command-line interface for managing versions of different packages.
```
Comment (kbdharun) [inline pages/common/mise.md:3]:
```suggestion
> Manage versions of different packages.
```
We prefer to not mention terms like "Common-line" at the start of definitions as by default we document command line commands only (if it's a GUI or TUI in that case we mention it in the description).

File: pages/common/mise.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Set global version for a package:
+
+`mise use --global {{name}}@{{version}}`
+
+- Set local version for a package:
+
+`mise use {{name}}@{{version}}`
+
+- Set environment variable in config:
```
Comment (kbdharun) [inline pages/common/mise.md:30]:
```suggestion
- Set environment variable in configuration:
```
We generally prefer having descriptive description, so I would suggest expanding it here, but if you are trying to imply a path with this example, then you can enclose it inside back ticks.

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**